test.sh

```diff
@@ -236,6 +238,15 @@ clean_up()
     fi
 }
 
+test_notebooks()
+{
+    echo "Testing notebooks"
+    for testfile in docs/Tutorial_*.ipynb
```

Comment:
```
for notebook in `grep ipynb docs/tutorials.rst | sed -e 's/^[ \t]*//'`
do
    jupyter nbconvert --to notebook --execute "$notebook"
    check_errs $?
done```

Reply:
So we are targetting the main tutorials. Cool.